<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTaskForm</class>
 <widget class="QDialog" name="NewTaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TitleLbl">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Title">
       <property name="styleSheet">
        <string notr="true">border-radius:4px;
padding:4px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DescLbl">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="Description">
       <property name="styleSheet">
        <string notr="true">border-radius:4px;
padding:4px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="EffortLbl">
       <property name="text">
        <string>Efort Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="EffortBox">
       <property name="styleSheet">
        <string notr="true">border-radius:4px;
padding:4px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PriorityLbl">
       <property name="text">
        <string>Priority</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="PriorityBox">
       <property name="styleSheet">
        <string notr="true">border-radius:4px;
padding:4px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Assignee</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="AssigneeBox">
       <property name="styleSheet">
        <string notr="true">border-radius:4px;
padding:4px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreateTaskBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(5, 109, 120);
color: rgb(255, 255, 255);
border-radius:8px;
padding-top:4px;
padding-bottom:4px;
padding-left:25px;
padding-right:25px;</string>
       </property>
       <property name="text">
        <string>Create Task</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
